helpers/ed.js

```diff
@@ -52,7 +54,13 @@ ed.makeKeypair = function(hash) {
  * @todo Add description for the params and the return value
  */
 ed.sign = function(hash, privateKey) {
-	return sodium.crypto_sign_detached(hash, privateKey);
+	if (!(hash instanceof Buffer))
```

Comment:
Why do we need that check here?

Reply:
It looks like a bug...

Reply:
I can recheck it but still type checking should be ok here to increase fail safety. At least I don't see why this check would cause any problems.

Reply:
It doesn't. I just wondering if that `sodium-native` library is really reliable.